.circleci/config.yml

```diff
@@ -780,6 +780,8 @@ jobs:
             other.test_full_js_library*
             core2.test_hello_world
             core2.test_fs_write_rawfs"
+            core2.*nodefs*
+            core2.*rawfs*
```

Comment:
Thats quite a few tests but maybe thats fine.  Can you remove  core2.test_fs_write_rawfs above?    

And you need to move the closing quote the last line.